tools/rc/usResourceCompiler.cpp

```diff
@@ -92,7 +90,7 @@ static char* get_error_str()
   LPVOID lpMsgBuf;
   DWORD dw = GetLastError();
 
-  FormatMessage(
+  FormatMessageA(
```

Comment:
This is a corner case; If ```FormatMessage``` fails, this function is left with a ```lpMsgBuf``` that is undefined and the pointer to potentially garbage memory is passed to ```va_start``` inside the ```exit_printf``` function. The end result to the user is going to be an unintelligible error message.

A similar problem was addressed as part of #33. I'd suggest doing something similar to fix this.